### PR TITLE
Eslint and Prettier setup

### PR DESCRIPTION
**Pull request by anndanielyan**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -4,8 +4,9 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Money Management Assistant</title>
     <link rel="stylesheet" href="./style/style.css" />
+    <script type="module" defer src="./script/script.js"></script>
```
Comment (ghost) [inline index.html]: @anndanielyan This part is not connected whit this ticket

File: index.html [hunk 2]
```diff
@@ -15,5 +16,9 @@ <h1 class="headline">Your Money Management Assistant</h1>
       <button class="sign-in btn">Sign In</button>
       <button class="sign-up btn">Sign Up</button>
     </article>
+    <section class="modal-window">
```
Comment (ghost) [inline index.html]: @anndanielyan This part is not connected whit this ticket

File: script/modal.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
+const signIn = `
```
Comment (ghost) [inline script/modal.js]: @anndanielyan This part is not connected whit this ticket

File: script/script.js [hunk 4]
```diff
@@ -0,0 +1,31 @@
+import { signIn } from './modal.js';
```
Comment (ghost) [inline script/script.js]: @anndanielyan This part is not connected whit this ticket

File: style/style.css [hunk 5]
```diff
@@ -52,6 +52,91 @@ body {
   color: #fff;
 }
 
+.close-modal {
```
Comment (ghost) [inline style/style.css]: @anndanielyan This part is not connected whit this ticket

**Review by ghost — CHANGES_REQUESTED** [review]
@anndanielyan Your ticket is connected with only eslint and prettier setup, please remove all other new added code, that is not connected to this PR, and leave ONLY the ticket requirements!

**Comment by anndanielyan** [comment]
> @anndanielyan Your ticket is connected with only eslint and prettier setup, please remove all other new added code, that is not connected to this PR, and leave ONLY the ticket requirements!

> @epamGevorg  Yes, I removed them

**Review by ghost — CHANGES_REQUESTED** [review]
.

**Review by ghost — CHANGES_REQUESTED** [review]
@anndanielyan can you don't speed up, and check your code functionality before opening new PR?
scripts are not connected, the branch name is not following the project naming.
Please, after corrections, check everything, and then open PR to save my and your time

**Comment by anndanielyan** [comment]
> @anndanielyan can you don't speed up, and check your code functionality before opening new PR? scripts are not connected, the branch name is not following the project naming. Please, after corrections, check everything, and then open PR to save my and your time

> @epamGevorg  I removed scripts connection, because as you said, it is not connected with this ticket. Have I change this branch's name? This real project with this tools, like git and github, linters, etc, is new for me, that's why I made mistakes like this. I am thankful for these new information and showing me right way. ))

**Pull request closed without merging**